src/components/CTCookieAgreement/index.js

```diff
@@ -9,14 +10,18 @@ export const AGREEMENT_ACCEPTED_KEY = 'class_transcribe_agreement_accepted';
 
 function CTCookieAgreement() {
   // Set true to false for testing purposes
-  const [dialogOpen, setDialogOpen] = useState(!(localStorage.getItem(AGREEMENT_ACCEPTED_KEY) === 'true'));
+  const [dialogOpen, setDialogOpen] = useState(
+    !(localStorage.getItem(AGREEMENT_ACCEPTED_KEY) === 'true')
+    &&
+    !user.isLoggedIn
+  );
```

Comment:
@aparekh5 Hi Aryaman, I made some changes here, I used the `user.isLoggedIn` to determine if a user is logged-in (logged-in <=> agreed the policies)